lib/assertions.js

```diff
@@ -462,6 +463,58 @@ export default class DOMAssertions {
     this.hasNoValue(message);
   }
 
+  /**
+   * Assert an [HTMLElement][] matching the `selector` is visible.
```

Comment:
can you add to the documentation what makes an element "visible"? for example if the browser is scrolled to a different part of the page does that mean that elements are not visible then?